analytics/analytics-ui/src/assets/defaultIcon.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is there a way we can make this an SVG file?

Reply:
Turned into SVG file.